pages/common/git-imerge.md

```diff
@@ -1,29 +1,29 @@
 # git-imerge
 
-> Perform a merge or rebase between two Git branches incrementally.
-> Conflicts between branches are tracked down to pairs of individual commits, to simplify conflict resolution.
+> Performs incremental merge or rebase between two Git branches.
+> Conflicts between are tracked down to pairs of individual commits, to simplify conflict resolution.
```

Comment:
I tried to remove "between branches", so that the phrase would read "Conflicts are tracked down to pairs of individual commits, to simplify conflict resolution.". Just to save some bytes.